nipype/interfaces/cat12/surface.py

```diff
@@ -0,0 +1,294 @@
+import os
+from pathlib import Path
+
+from traits.trait_base import _Undefined
+
+from nipype.interfaces.base import File, InputMultiPath, TraitedSpec, traits
```

Comment:
We have the `isdefined` function.

```suggestion
from nipype.interfaces.base import File, InputMultiPath, TraitedSpec, traits, isdefined
```